package.json

```diff
@@ -9,10 +9,15 @@
   "keywords": [
     "react",
     "rxjs",
-    "props"
+    "props",
+    "proposal-decorators"
   ],
   "author": "mcreader215@gmail.com",
   "license": "MIT",
+  "engines": {
```

Comment:
peerDependencies?

Reply:
Research and examine result:
Using exact paths reduces the bundle size by 551.2 KiB.